app/routes/News.tsx

```diff
@@ -0,0 +1,58 @@
+import { NavLink, Outlet } from '@remix-run/react'
+
+export default function NewsPage() {
+	return (
+		<main className="flex flex-col py-24">
+			<div className="container py-16">
+				<h1 className="text-3xl font-bold text-gray-800">News</h1>
+				<h2 className="text-4xl">
+					Select a news category from the choices above...
+				</h2>
+
+				<div className="flex space-x-4 py-4">
```

Comment:
Move this section with your NavLink components into a new file called `news._index.tsx`. This will ensure the links remain at the top of the page when users navigate to the different article category pages.